src/jpegxl.imageio/jxloutput.cpp

```diff
@@ -412,6 +519,9 @@ JxlOutput::save_image(const void* data)
 
     DBG std::cout << "data = " << data << " size = " << size << "\n";
 
+    // Write EXIF info
+    bool metadata_success = save_metadata(m_spec, m_encoder);
```

Comment:
Clang gives a warning about the fact that you don't ever use `metadata_success`. Should you actually handle any errors preparing the metadata?

Reply:
Hm. I'll add something.